apps/charge-accounts-service/src/auth/auth.controller.ts

```diff
@@ -28,4 +33,21 @@ export class AuthController {
     const user = await this.usersService.findOneByAuth0Id(id)
     return { id: user?.id }
   }
+
+
+  /**
+   * Validate operator
+   * @param authOperatorDto
+   * @returns the new simple JWT
+   */
+  @Post('/validate')
```

Comment:
Here i think you can name endpoint as /operator to make clear why what is this endpoint for

Reply:
@NikolayRn sure, I'll update it. Btw I think we can keep it as is to provide a more general functionality for verifying signatures and generating simple JWTs for future use cases. Wdyt?

Reply:
I think endpoint`s name "validate" might not be the best choice because it doesn't clearly convey the purpose of the endpoint, which is to generate a JWT for operators. A more descriptive name like "generateOperatorJwt" or "operatorJwt" would better communicate the functionality of the endpoint. @MusabShakeel576

Reply:
@NikolayRn renamed /validate to /operator